SwiftUIBasics/Views/RatingView.swift

```diff
@@ -1,17 +1,32 @@
-//
-//  RatingView.swift
-//  SwiftUIBasics
-//
-//  Created by Diplomado on 09/12/23.
-//
+
 
 import SwiftUI
 
-struct RatingView: View {
-    var body: some View {
-        Text(/*@START_MENU_TOKEN@*/"Hello, World!"/*@END_MENU_TOKEN@*/)
-    }
-}
+ struct RatingView: View {
+     @State private var rating = 0
+     private var starColor = Color(hue: 0.1333, saturation: 0.94, brightness: 0.97)
```

Comment:
I like the idea of custom colors, but this will be better if you put the color in the assets